google-cloud-storage/src/main/java/org/jclouds/googlecloudstorage/features/ObjectApi.java

```diff
@@ -199,7 +198,7 @@ GoogleCloudStorageObject simpleUpload(@PathParam("bucket") String bucketName, @H
    @Named("Object:delete")
    @DELETE
    @Path("storage/v1/b/{bucket}/o/{object}")
-   @Fallback(TrueOnNotFoundOr404.class)
+   @Fallback(FalseOnNotFoundOr404.class)
```

Comment:
Definitely, can you either modify an existing test or add a new test?  We need not check at all call sites, but at least the basic delete test should do this.  Thanks!